content/en/monitors/monitor_types/forecasts.md

```diff
@@ -15,91 +15,126 @@ further_reading:
+### Define the metric
 
-Navigate to the [Monitor page][1] for **Forecast Alerts**. Then fill out the **Define the metric** section just as you would for any other metric monitor.
+Any metric currently reporting to Datadog is available for monitors. For more information, see the [Metric Monitor][2] page.
 
-{{< img src="monitors/monitor_types/forecasts/alert_conditions.png" alt="alert conditions" responsive="true" style="width:80%;">}}
+After defining the metric, the forecast monitor provides two preview graphs in the editor:
+{{< img src="monitors/monitor_types/forecasts/editor_graphs.png" alt="Editor graphs" responsive="true" style="width:95%;">}}
 
-There are three required options for setting up a forecast alert:
+* The **Historical View** lets you explore the past metric data at different time scales.
+* The **Evaluation Preview** shows a combination of historical and predicted metric data.
 
-* The threshold at which an alert is triggered. For a metric like `system.disk.in_use`, set to 1.0, whereas for a metric like `system.mem.pct_usable`, set to 0.0. A [recovery threshold][2] is also required.
-* The condition on which an alert is triggered. For a metric like `system.disk.in_use`, set to "above or equal to", whereas for a metric like `system.mem.pct_usable`,  set to "below or equal to".
-* Control how far in advance you would like to be alerted before your metric hits its critical threshold.
+### Set alert conditions
 
-{{< img src="monitors/monitor_types/forecasts/alert_advanced.png" alt="alert advanced" responsive="true" style="width:80%;" >}}
+* Trigger an alert when the edge of the forecast confidence bounds goes `above` or `below`
+* the threshold within the next `24 hours`, `1 week`, `1 month`, etc.
+* Alert threshold: >= `<NUMBER>`
+* Alert [recovery threshold][3]: < `<NUMBER>`
 
-Datadog automatically sets the **Advanced** options for you by analyzing your metric. Note that any changes in the **Define the metric** section could change the advanced options.
+#### Advanced Options
 
-* You can change the forecasting algorithm to be used here. See the [Forecast algorithms](#forecast-algorithms) section for tips on how to choose the best algorithm for your use case. Each algorithm also has additional settings is described in the next section.
-* Datadog recommends using larger intervals between points to avoid having noise influence the forecast too much.
-* The number of deviations controls the width of the range of forecasted values. A value of 1 or 2 should be large enough to forecast most "normal" points accurately.
+Datadog automatically analyzes your chosen metric and sets several parameters for you. However, the options are available to edit under **Advanced Options (auto)**:
```

Comment:
```suggestion
Datadog automatically analyzes your chosen metric and sets several parameters for you. However, the options are available to edit under **Advanced Options**:
```

The "(auto)" is only there during initial monitor creation and if the user hasn't already edited any options.